e -iq

```diff
@@ -0,0 +1,294 @@
+[33mcommit 9eddb12ea645edf8b86cec0f7e9b6b4ee006f202[m[33m ([m[1;36mHEAD -> [m[1;32mfeature/search[m[33m)[m
```

Comment:
Why is this file being added ?

Could you remove it if it is unintentionally added ?

Reply:
done